plugins/modules/foreman_scap_content.py

```diff
@@ -129,11 +133,20 @@ def main():
             if not entity and 'scap_file' not in module_params:
                 module.fail_json(msg="Content of scap_file not provided. XML containing SCAP content is required.")
 
+            if 'scap_file' in module_params:
+                with open(module_params['scap_file']) as input_file:
```

Comment:
Shouldn't this be reading with "rb"?

Reply:
given it's xml, I can see r, but not b :)

Reply:
you may not need the `.encode("utf-8")` below.

Reply:
Also (and i know i get confused every time), this file needs to live on the machine where the module is executed, right?

Reply:
Yes.

Reply:
`hashlib.sha256` needs a bytestream, not a string, but the API needs a string, not a bytestream.

So I pretend to read text, and push bytestream to hashlib. :boom:

Reply:
Convinced.